src/server/bots/bridgeless/bridgelessBot.ts

```diff
@@ -0,0 +1,88 @@
+import type { Request, Response } from 'express'
+
+import type { Autobot, AutobotEngineArgs } from '../../types'
+import {
+  createBridgelessDoc,
+  createCouchConnection,
+  deleteBridgelessDoc,
+  getAllBridgelessDocs,
+  updateBridgelessDoc
+} from './databaseService'
+import { chainUtils, submitBridgelessDeposit } from './txidSubmissionService'
+import { asBridgelessSubmission } from './types'
+
+export async function bridgelessBotEngine({
+  log
+}: AutobotEngineArgs): Promise<void> {
+  const db = createCouchConnection()
+
+  log('Processing all txids...')
+  const configs = await getAllBridgelessDocs(db)
+  if (configs == null) {
+    log('No txids to process')
+    return
+  }
+
+  for (const [chainId, documents] of Object.entries(configs)) {
+    log(`Processing ${documents.length} txids for chain ${chainId}`)
+    const chainHeight = await chainUtils[chainId].getChainHeight()
```

Comment:
getChainHeight could throw. catch it so you can at least process the other chains